projects/eval-pqmon/Makefile

```diff
@@ -0,0 +1,22 @@
+BASIC_EXAMPLE = y
+IIOD=y
+
+HEAP_SIZE=0x80000
+STACK_SIZE=0x20000
+
+# Define PQM libray path here:
+PQLIB_PATH=''
```

Comment:
the build should be invokable with `make PQLIB_PATH=/my/path`  or similar without editing this file